lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -38,7 +37,7 @@ def self.run(params)
         if app_id.nil?
           UI.crash!(ErrorMessage::MISSING_APP_ID)
         end
-        release_id = fad_api_client.upload(app_id, binary_path)
+        release_id = fad_api_client.upload(app_id, binary_path, platform.to_s)
```

Comment:
nit: is the `to_s` necessary?

Reply:
Yes because if not then it passes in a symbol and when you try to set it in the header it gives you an error. 
<img width="1776" alt="Screen Shot 2020-08-06 at 10 12 28 AM" src="https://user-images.githubusercontent.com/32231345/89542199-57d3fe80-d7cd-11ea-810b-21e4694b9f28.png">